Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -1055,11 +1055,11 @@ public async Task<BoolResult> TrimBulkAsync(OperationContext context, IReadOnlyL
                 () =>
                 {
                     var effectiveLastAccessTimes = new List<ContentHashWithLastAccessTimeAndReplicaCount>();
-                    int replicaCount = 1;
 
                     foreach (var contentHash in contentHashes)
                     {
                         DateTime? effectiveLastAccessTime = null;
+                        int replicaCount = 1;

```

Comment:
nit: effectiveReplicaCount? to follow the same naming scheme to effectiveLastAccessTime?